disperser/cmd/controller/main.go

```diff
@@ -34,6 +35,11 @@ var (
 	version   string
 	gitCommit string
 	gitDate   string
+
+	controllerReadinessProbePath string        = "/tmp/controller-ready"
+	controllerHealthProbePath    string        = "/tmp/controller-health"
+	controllerMaxStallDuration   time.Duration = 240 * time.Second
+	controllerLivenessChan                     = make(chan time.Time, 1)
```

Comment:
Doesn't seem like we ever send a liveness message to this channel in the code. See https://github.com/Layr-Labs/eigenda/blob/a1fceb9e015325746d5db9e14b5fbc515b731ca1/disperser/batcher/batcher.go#L450 for an example